core/src/main/scala/org/apache/spark/TaskContext.scala

```diff
@@ -126,14 +134,14 @@ abstract class TaskContext extends Serializable {
   }
 
   /**
-   * Adds a listener to be executed on task failure.
-   * Operations defined here must be idempotent, as `onTaskFailure` can be called multiple times.
```

Comment:
This was disabled in https://github.com/apache/spark/pull/11504. So the comment does not make sense anymore.